integrationTests/sync/edgeCases_test.go

```diff
@@ -20,6 +21,8 @@ func TestSyncMetaNodeIsSyncingReceivedHigherRoundBlockFromShard(t *testing.T) {
 		t.Skip("this is not a short test")
 	}
 
+	_ = logger.SetLogLevel("*:INFO,*:DEBUG,*:TRACE")
```

Comment:
setLogLevel should be just one of them. This way the last one is set.

Reply:
done.